src/Pages/Nodes.razor

```diff
@@ -205,9 +215,48 @@
         }
     }
 
+    private string _NodesState(Node node)
+    {
+       return node.IsNodeEnabled ? "Disable Node" : "Enable Node";
+    }
+
+    private async Task NodeManager(Node node)
+    {
+        try
+        {
+          
+            if (node.IsNodeEnabled)
+            {
+                await StopJob(node);
+            }
+            else
+            {
+                await CreateJobs(node);
+            }
+            node.IsNodeEnabled = !node.IsNodeEnabled;
```

Comment:
this is confusing. Please first update the state of the boolean and then do the if statement

Reply:
🤔 While this might not be overly confusing, I personally don't find it to be a significant issue.